src/lib/components/forms/form/form-components/text-input.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { TerraFormComponentBaseInterface } from './terra-form-component-base.interface';
+
+export interface TextInputInterface extends TerraFormComponentBaseInterface {
+    isPassword: boolean;
+    isIban: boolean;
+    isReadonly: boolean;
+    minValue: number;
+    maxValue: number;
```

Comment:
eher min- und maxLength, oder?

Reply:
kann ich auch machen. Beim Number Input hießen sie auch nur minValue & maxValue

Reply:
ja aber das sind halt zwei verschiedene Sachen.
min- und maxValue überprüfen ob die eingegebene Zahl in den angegeben Grenzen liegt.
min- und maxLength dagegen überprüfen, ob die eingegebene Anzahl an Zeichen in den angegebenen Grenzen liegt.
Ersteres macht bei nem text input keinen Sinn. Zweiteres macht bei einem number input keinen Sinn.

Reply:
Okay ja stimmt 🙈  pass ich an